### PR TITLE
[Doc] Add charts to examples

### PR DESCRIPTION
**Pull request by nicolas-dufour**
## Description

This PR adds performance charts to the examples README.

## Motivation and Context

The motivation is to allow the reviewer to be able to have an idea of the performances of the library

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [x] New feature (non-breaking change which adds core functionality)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/facebookresearch/rl/blob/main/CONTRIBUTING.md) guide (**required**)

### CODE REVIEW

File: examples/EXAMPLES.md [hunk 1]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
 ```
 python sac frames_per_batch=63
 ```
+# Results
+
+## Half Cheetah
+Here we can see some results for the SAC and REDQ algorithm on the Half Cheetah environement.
```
Comment (vmoens) [inline examples/EXAMPLES.md:26]: What about discarding this sentence, which does not add much?

File: examples/EXAMPLES.md [hunk 2]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
 ```
 python sac frames_per_batch=63
 ```
+# Results
+
+## Half Cheetah
```
Comment (vmoens) [inline examples/EXAMPLES.md:25]: Let's write the proper name, ie "Gym's HalfCheetah-v4"

File: examples/EXAMPLES.md [hunk 3]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+```
+python examples/sac/sac.py env_name="HalfCheetah-v4" env_task="" env_library="gym"
+```
+
+``` 
+python examples/redq/redq.py env_name="HalfCheetah-v4" env_task="" env_library="gym"
+```
+
+
+## Cheetah
```
Comment (vmoens) [inline examples/EXAMPLES.md:42]: Same, "dm_control's `cheetah`-`run` task"

File: examples/EXAMPLES.md [hunk 4]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+Here we can see some results for the SAC and REDQ algorithm on the Cheetah environement.
```
Comment (vmoens) [inline examples/EXAMPLES.md:43]: Same I don't think it adds much

File: examples/EXAMPLES.md [hunk 5]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+
+```
+python examples/sac/sac.py env_name="cheetah" env_task="run" env_library="dm_control"
+```
+
+``` 
+python examples/redq/redq.py env_name="cheetah" env_task="run" env_library="dm_control"
+```
+
+## Ant
```
Comment (vmoens) [inline examples/EXAMPLES.md:58]: same

File: examples/EXAMPLES.md [hunk 6]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+Here we can see some results for the SAC and REDQ algorithm on the Ant environement.
```
Comment (vmoens) [inline examples/EXAMPLES.md:59]: same

File: examples/EXAMPLES.md [hunk 7]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+
+```
+python examples/sac/sac.py env_name="Ant-v4" env_task="" env_library="gym"
+```
+
+``` 
+python examples/redq/redq.py env_name="Ant-v4" env_task="" env_library="gym"
+```
+
+## Walker2D
```
Comment (vmoens) [inline examples/EXAMPLES.md:74]: Same

File: examples/EXAMPLES.md [hunk 8]
```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
...
+Here we can see some results for the SAC and REDQ algorithm on the Walker2D environement.
```
Comment (vmoens) [inline examples/EXAMPLES.md:75]: same

File: examples/EXAMPLES.md (hunk 1 above)
Comment (nicolas-dufour) [reply examples/EXAMPLES.md:26]: I've changed it such as it's only at the begining of the results section and not in every example

**Comment by vmoens** [comment]
Can we change "Then, go to the directory that interests you and run" which sounds a bit frenchy to "Scripts can be run from the directory of interest using:"?

**Review by vmoens — APPROVED** [review]

**Pull request merged**